medicines/web/components/search-results/index.tsx

```diff
@@ -239,7 +237,7 @@ const SearchResults = (props: {
                   </dt>
                   <dd className="right">
                     {drug.product != null ? (
-                      <a href={drug.url}>
+                      <a href={drug.url} download="x">
```

Comment:
Not sure why but this `download` attribute doesn't seem to do anything for me locally. I just navigate to the PDF as normal.

Reply:
good catch! I was testing out the HTML5 download attribute